src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorWithComments.java

```diff
@@ -69,42 +69,42 @@
 
 
 /* violation */
-// violation above
+ // violation above ''/*' has more than 1 empty lines before.'
 import java.lang.Number;
 
 
-import java.lang.Runnable; // violation
+import java.lang.Runnable; // violation ''import' has more than 1 empty lines before.'
 import java.lang.Thread;
 
 // no violation
 
 
-// violation
+// violation ''//' has more than 1 empty lines before.'
 import java.lang.StringBuilder;
 
 
-/*
- * // violation above
+/* // violation ''/*' has more than 1 empty lines before.'
+ *
```

Comment:
The violation message is interpreted as a regular expression, you need to escape `*` as it is a special character. You may escape using `\` or may use https://stackoverflow.com/questions/30776860/regular-expression-q-e